frontend/__tests__/routes/_oh.test.tsx

```diff
@@ -1,4 +1,5 @@
-import { afterEach, beforeAll, describe, expect, it, vi } from "vitest";
+import { afterEach, beforeAll, beforeEach, describe, expect, it, vi } from "vitest";
```

Comment:
Is beforeEach used here

Reply:
Nope. I wonder if we're not linting these files..